packages/now-cli/src/commands/deploy/legacy.ts

```diff
@@ -852,7 +852,7 @@ async function sync({
     const { url } = now;
     const dcs =
       deploymentType !== 'static' && deployment.scale
-        ? chalk` ({bold ${Object.keys(deployment.scale).join(', ')})`
+        ? chalk(`${chalk.bold(Object.keys(deployment.scale).join(', '))}`)
```

Comment:
```suggestion
        ? chalk` ({bold ${Object.keys(deployment.scale).join(', ')}})`
```

Reply:
Why keep the template function? I think it's just more confusing and error prone

Reply:
I don't think the outer `chalk()` function is necessary then, just use a JS string template and `chalk.bold()` inside.

Reply:
I think this matches the previous output:

```suggestion
        ? ` (${ chalk.bold(Object.keys(deployment.scale).join(', ')) })`)
```